CryptoTechReminderSystem/UseCase/GetProjectManagersWithOpenTimeEntries.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.DomainObject;
+using CryptoTechReminderSystem.Gateway;
+
+
+namespace CryptoTechReminderSystem.UseCase
+{
+    public class GetProjectManagersWithOpenTimeEntries : IGetLateBillablePeople
+    {
+        private readonly IHarvestBillablePersonRetriever _harvestBillablePersonRetriever;
+        private readonly ITimeSheetRetriever _harvestTimeSheetRetriever;
+        private readonly ISlackBillablePersonRetriever _slackBillablePersonRetriever;
+        private readonly IClock _clock;
+        
+        public GetProjectManagersWithOpenTimeEntries(ISlackBillablePersonRetriever slackBillablePersonRetriever, IHarvestBillablePersonRetriever harvestBillablePersonRetriever, ITimeSheetRetriever harvestTimeSheetRetriever, IClock clock) {
+            _slackBillablePersonRetriever = slackBillablePersonRetriever;
+            _harvestBillablePersonRetriever = harvestBillablePersonRetriever;
+            _harvestTimeSheetRetriever = harvestTimeSheetRetriever;
+            _clock = clock;
+        }
+        
+        public GetLateBillablePeopleResponse Execute() 
+        {
+            var dateFrom = GetStartingDate(_clock.Now());
+            var dateTo = GetEndingDate(_clock.Now());
+            
+            var timeSheets = _harvestTimeSheetRetriever.RetrieveTimeSheets(dateFrom, dateTo);
+            var billablePeople = _harvestBillablePersonRetriever.RetrieveBillablePeople();
+            var slackGetBillablePeopleResponse = _slackBillablePersonRetriever.RetrieveBillablePeople();
+
+            return ProjectManagersWithOpenTimeEntries(timeSheets, billablePeople, slackGetBillablePeopleResponse);
+        }
+
+        private GetLateBillablePeopleResponse ProjectManagersWithOpenTimeEntries(IList<DomainObject.TimeSheet> timeSheets, IList<DomainObject.HarvestBillablePerson> billablePeople, IList<DomainObject.SlackBillablePerson> slackGetBillablePeopleResponse) {
+            var getProjectManagersWithOpenTimeEntriesResponse = new GetLateBillablePeopleResponse
+            {
+                BillablePeople = new List<GetLateBillablePeopleResponse.LateBillablePerson>()
+            };
+
+            foreach (var timeSheet in timeSheets) {
```

Comment:
(This is real nitpicking) I would move this bracket down to a new line to match the rest of the code in the project.